lib/osf-components/addon/components/institution-select-list/template.hbs

```diff
@@ -11,19 +11,9 @@
             <li data-test-institution={{institution.id}}>
                 <label local-class='label'>
                     {{#let (contains institution @manager.affiliatedList) as |isAffiliated|}}
-                        <OsfButton
-                            data-test-institution-button='{{if isAffiliated 'remove' 'add'}}-{{institution.id}}'
-                            data-analytics-name={{if isAffiliated 'Remove' 'Add'}}
-                            local-class='addRemoveButton'
-                            @type='link'
-                            @disabled={{@manager.save.isRunning}}
-                            @onClick={{if isAffiliated (action @manager.removeInstitution institution) (action @manager.addInstitution institution)}}
-                        >
-                            <FaIcon
-                                local-class='institutionIcon'
-                                @icon={{if isAffiliated 'minus' 'plus'}}
-                            />
-                        </OsfButton>
+                        {{input (html-attributes data-test-institution-button=(concat (if isAffiliated 'remove' 'add') '-' institution.id ) data-analytics-name=(if isAffiliated 'Remove' 'Add'))
```

Comment:
One of these days we need one of us to get prettier (or something like it) set up, but in the meanwhile, would you mind splitting long lines along consistent boundaries? If a list of args is too long to fit on one line, then put a line break before *every* arg, not just some of them. It helps a brain tokenize accurately.

```hbs
{{input
    (html-attributes
        data-test-institution-button=(concat (if isAffiliated 'remove' 'add') '-' institution.id )
        data-analytics-name=(if isAffiliated 'Remove' 'Add')
    )
    type='checkbox'
    checked=isAffiliated
    disabled=@manager.save.isRunning
    change=(action @manager.toggleInstitution institution)
}}
```

Reply:
(That `(concat (if isAffiliated 'remove' 'add') '-' institution.id )` would be cleaner as a computed property, but the actual problem is that the tests were written to expect a concatenated blob instead of just `id`, since they could use the `checked` attr to tell the state. Probably beyond the scope of this ticket to fix that, so I think it's fine as-is.)